asreview/review/simulate.py

```diff
@@ -77,7 +77,8 @@ def sample_prior_knowledge(
 
 
 def naive_prior_knowledge(labels):
-    """Select top records until the first 0 and 1 are found.
+    """Select records from the dataset as priors in the order they were
+    provided until the first 0 and 1 are encountered
```

Comment:
```suggestion
    """Select top records until the first 0 and 1 are found.
```

Reverting this to merge this PR quickly. The first line of the docstring should be a single line (PEP257).